src/components/input/Input.styled.ts

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+import { themes } from "../../themes";
+
+export const InoputContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 2em 0 2em 0;
+	gap: 1rem;
+	width: 90%;
+`;
+
+export const Input = styled.input`
+	height: 4em;
+	margin: 0 auto;
+	border: 1px solid ${themes.light.dimGray};
+	color: ${themes.light.background};
+	border-radius: 2em;
+	text-align: center;
+	font-size: 1em;
+	width: 100%;
+
+	@media (max-width: 480px) {
+		height: 3em;
+	}
+
+	&::placeholder {
+		color: ${themes.light.background};
+		opacity: 0.6;
+	}
+`;
+
+export const Button = styled.button`
+	margin: 0 auto;
+	height: 4em;
```

Comment:
Ustawianie height dla buttonu to błąd.
Wysokośc buttona powinna sama się ustalać poprzez wartości:
padding + line height + font size.

Ustalanie wysokosci na sztywno rpowadzi do problemów, np wtedy gdy jakiś ekran wyświetli button złamany na dwie linijki, wtedy tekst będzie niewidoczny